### DISCUSSION TITLE
Check for package updates

### DISCUSSION THREAD

**Discussion by helderberto** [npm/feedback#393] [CLI] [answered]
I would like to ask if it's possible or in the roadmap to add something like yarn upgrade approach i.e `yarn upgrade-interactive --latest` or like this package [npm-check-updates](https://github.com/raineorshine/npm-check-updates).

It can be nice and very helpful something like `npm upgrade-interactive --latest`.

What do you think about that?

**Comment by MylesBorins** [comment, marked as answer]
Hey @helderburato for CLI specific feature request we ask that you open an issue over at https://github.com/npm/rfcs outlining the feature you have in mind.

Thanks.

**Reply by helderberto** [reply]
Thanks for the reply. I’ll create the RFC then.

**Reply by manovotny** [reply]
For those looking for the RFC for this, it appears to be this: https://github.com/npm/rfcs/issues/137
